reproducible/mknodeimage.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -ex
+
+# usage() {
+#   echo "Usage: $0 [SNAPSHOT] [VARIANT] [DIST]"
+#   echo
+#   echo "[SNAPSHOT]: The debian snapshot datetime to use."
+#   echo "[VARIANT]: The debian variant to use."
+#   echo "[DIST]: The debian dist to use."
+#   echo
+#   exit 1
+# }
+
+# if [ $# -ne 1 ]; then
+#     usage
+# fi
+
+WORKDIR="/workspace/nodejs_build"
+OUTPUTDIR="$WORKDIR/nodejs"
+mkdir -p "$OUTPUTDIR"
+cd "$WORKDIR"
+
+# TODO(aaron-prindle) make sure the rootfs is structured properly (isn't rn)
+curl -o node-v6.11.4.tar.gz https://nodejs.org/dist/v6.11.4/node-v6.11.4.tar.gz
```

Comment:
Could we grab this via WORKSPACE instead?

Reply:
done